mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -87,7 +87,7 @@
     <!-- Browser menu button that creates a new tab -->
     <string name="browser_menu_new_tab">New tab</string>
     <!-- Browser menu button that saves the current tab to a collection -->
-    <string name="browser_menu_save_to_collection">Save to Collection</string>
+    <string name="browser_menu_save_to_collection_2">Save to collection</string>
```

Comment:
@BranescuMihai

For future reference: changes to capitalization don't require using a new ID. Each language has its rules, no point in losing translations.
https://developer.mozilla.org/en-US/docs/Making_String_Changes

Reply:
@flodolo thanks for the info. In this scenario, I think it wasn't just a capitalisation mistake (since 'Collection' would refer to the actual feature), but rather a UX/UI decision, so I assumed it needs to be changed in every language (for example, en-rCA and en-rGB)

But yes, I'll keep that in mind next time!